src/common/logger.py

```diff
@@ -28,7 +28,7 @@
         },
         'standard': {
             'format': '%(asctime)s %(levelname)-10s%(message)s',
-            'datefmt': '%y/%m/%d %H:%M:%S',
+            'datefmt': '%y-%m-%d %H:%M:%S',
```

Comment:
```suggestion
            'datefmt': '%Y-%m-%d %H:%M:%S',
```
#431 started as "it needs to have year on 4 chars instead of 2", 'Y' is needed in the format

Reply:
My fault, forgot to apply ALL the changes.